lib/unicore/mktables

```diff
@@ -871,6 +871,15 @@ push @tables_that_may_be_empty, 'Grapheme_Cluster_Break=Prepend'
 push @tables_that_may_be_empty, 'Canonical_Combining_Class=CCC133'
                                                     if $v_version ge v6.2.0;
 
+# These properties of Egyptian hieroglyphs are not handled by Perl.  Their
+# intended audience is only specialist Egyptologists
+push @tables_that_may_be_empty, qw(kEH_Cat kEH_Desc kEH_HG kEH_IFAO
```

Comment:
What does it do? And why would we not want to support it?

Reply:
I think it important to get the next Perl version shipped with the latest Unicode release, and I think in order to do this, it has to be in the the upcoming development release due out in the next day or two.  Getting this to work in time is lower priority than getting the rest to work in time.  These could be legally fixed in the next development release next month.    And since the bus factor for getting it in is 1, I don't think the comments should promise anything.

For information, see https://www.unicode.org/reports/tr57/tr57-3.html

Reply:
"are not handled by Perl" is ambiguous. It could be read as "are not to be handled" (so don't add them) or "are not handled *yet*".

Reply:
Changed to `These properties of Egyptian hieroglyphs are not yet handled by Perl.  Their`